dev-docs/bidders/smartx.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: bidder
+title: smartx
+description: Prebid smartx Bidder Adaptor
```

Comment:
Typo: Adapt*ar*

Reply:
Can it be that both of us are wrong?
https://github.com/prebid/Prebid.js/tree/master/modules
All modules here are named with "e" in Adapt**e**r.

Reply:
Indeed :joy: Mixing german and english makes my english so much worst :grimacing: